public/js/src/articleModel.js

```diff
@@ -0,0 +1,12 @@
+(function(exports) {
```

Comment:
So you totally have an article class. You should use this to store information in the article list.